src/Microsoft.VisualStudio.Editors/PropPages/ApplicationPropPage.vb

```diff
@@ -611,6 +611,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                         If Trim(Win32ResourceFile.Text).Length = 0 Then
                             message = My.Resources.Microsoft_VisualStudio_Editors_Designer.PropPage_NeedResFile
                             Return ValidationResult.Warning
+                        ElseIf Not Win32ResourceFile.Text.IndexOfAny(Path.GetInvalidPathChars()) = -1 Then
+                            message = My.Resources.Microsoft_VisualStudio_Editors_Designer.PropPage_InvalidCharactersInFilePath
```

Comment:
Is the set of characters returned by `Path.GetInvalidPathChars()` generally small? If so, consider updating the message to list the characters that may not be used.

Reply:
It looks to be around 36 characters. Do you think that is too many to display?

Reply:
Or maybe the error could show the invalid characters that are being used?

Reply:
@etbyrd I agree. Showing which characters the user must remove in order for the value to be accepted seems like the sweet spot here.

Reply:
Hmm. 36 is too many. In that case, either include just the first invalid character in the message, or all the invalid characters. Just the first will be easier and entirely sufficient, I think.

Reply:
Showing all of them will involve de-duplication and figuring out how to localize a list of arbitrary length. I'm not convinced that level of effort is worth it.

Reply:
I agree. I think just showing the first invalid character would be enough here.

Reply:
I added the first invalid character to the message, what do you think?
![Screenshot (8)](https://user-images.githubusercontent.com/8518253/66091495-dd977580-e53c-11e9-94b3-29a9eb2c50cf.png)